### ISSUE TITLE
Go Through Code and Fix Warnings

### ISSUE THREAD

**Issue by synnestorm** [synnestorm/semester-project-museum#21]
Tested all pages through WC3 Validator:

- [x] Warning on all pages: Possible misuse of aria-label (this is the hamburger menu)
- [x] Warning on index.html: Section lacks heading. Consider using h2-h6 elements to [add identifying headings to all sections](https://www.w3.org/wiki/HTML/Usage/Headings/Missing), or else use a div element instead for any cases where no heading is needed

**Comment by synnestorm** [comment]
WC3 Testing Complete: all the pages have no errors or warnings to show, according to https://validator.w3.org/

The problem: The aria-label was misplaced inside the i tag, when it should have been inside the label tag
